packages/react-next/src/components/SwatchColorPicker/ColorPickerGridCell.base.tsx

```diff
@@ -1,157 +1,135 @@
 import * as React from 'react';
 import { ITheme, mergeStyleSets } from '../../Styling';
-import { classNamesFunction, memoizeFunction } from '../../Utilities';
-import { getColorFromString } from 'office-ui-fabric-react/lib/utilities/color/getColorFromString';
-import { GridCell, IGridCellProps } from '../../utilities/grid/index';
+import { classNamesFunction } from '../../Utilities';
+import { getColorFromString } from 'office-ui-fabric-react/lib/Color';
+import { GridCell } from '../../utilities/grid/GridCell';
```

Comment:
Guessing you're planning to do the ButtonGrid rename here in a subsequent PR? (that's fine)

Reply:
That's what I was planning. This PR is already quite large so for readability it would probably be best to rename it to ButtonGrid after the Grid conversion.